src/_helpers/fonts.js

```diff
@@ -0,0 +1,17 @@
+const getLineHeightFromTextSize = (textSize, theme) => {
```

Comment:
rename to `getLineHeight`
Also rearrange the arguments in colors we have theme first and then the color let's follow the same here

Reply:
done